Source/AlchemyVisionV1/AlchemyVision.swift

```diff
@@ -160,6 +162,24 @@ public class AlchemyVision {
         failure: (NSError -> Void)? = nil,
         success: ImageLink -> Void)
     {
+        // encode html document
+        guard let htmlEncoded = html.stringByAddingPercentEncodingWithAllowedCharacters(unreservedCharacters) else {
+            let failureReason = "Failed to percent encode HTML document."
```

Comment:
This works, however I wonder how often html will actually be passed in as a String. Would it be better to take in a NSURL of a document, create a String from that, and then encode?